src/EventGrid/EventGrid/Models/PSTopic.cs

```diff
@@ -30,6 +30,81 @@ public PSTopic(Topic topic)
             this.ProvisioningState = topic.ProvisioningState;
             this.Tags = topic.Tags;
             this.Endpoint = topic.Endpoint;
+            this.InputSchema = topic.InputSchema;
```

Comment:
is it necessary to have topic?.InputSchema?

Reply:
same as above.